### PR TITLE
Added Smooth ScrollToTop Functionality

### PR DESCRIPTION
**Pull request by ZubairImtiaz3**
Scroll Smooth Behaviour To The Top Is As Follow After My Changes

[Contribute-To-This-Project.webm](https://user-images.githubusercontent.com/106450046/203980836-b4f4ba1c-4991-4f4c-bf1e-9f0cb0ff8466.webm)

### CODE REVIEW

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@ZubairImtiaz3, remove the conflict comments such as Head, master and extra spaces. and kindly remove your changes with fonts. if you want to change anything else kindly describe the changes with a valid reason. Thanks!

for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)

**Comment by ZubairImtiaz3** [comment]
> @ZubairImtiaz3, remove the conflict comments such as Head, master and extra spaces. and kindly remove your changes with fonts. if you want to change anything else kindly describe the changes with a valid reason. Thanks!
> 
> for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)



> @ZubairImtiaz3, remove the conflict comments such as Head, master and extra spaces. and kindly remove your changes with fonts. if you want to change anything else kindly describe the changes with a valid reason. Thanks!
> 
> for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)

I'm new to the version control system that's the reason i didn't notice those changes, now i have removed the merge conflict comments.

File: assets/style.css [hunk 1]
```diff
@@ -19,6 +19,15 @@ body {
...
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
+/* FONTS */
+h1{
+  font-family: 'Domine', serif;
+}
```
Comment (TenTraicion) [inline assets/style.css:29]: @ZubairImtiaz3 why did you add the fonts?

Comment (ZubairImtiaz3) [reply assets/style.css:29]: The font target the main heading which is Contribute to this project. This was added long ago by me n that pull request was merged.
I didn't add it again but dunno why it is added.

I think there i might not be using the version control system correctly

Comment (TenTraicion) [reply assets/style.css:29]: @ZubairImtiaz3, kindly remove those lines that are unnecessary...

Comment (ZubairImtiaz3) [reply assets/style.css]: Ok

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@ZubairImtiaz3, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.
